packages/dara-components/changelog.md

```diff
@@ -2,6 +2,11 @@
 title: Changelog
 ---
 
+## NEXT
```

Comment:
Changelog should be updated to the correct version number

Reply:
Isn't it updated as part of CI?

Reply:
Same as in the ui repo, the CI does not modify any files as it does not have the privileges to commit changes.

Reply:
Right, it only updates it in the artifact